src/components/styled-atlaskit/Button/Button.scss

```diff
@@ -76,29 +80,138 @@ button.styled-atlaskit-button {
       &:hover {
         background-color: $button-backgroundColor--subtle--hovered;
       }
-      &:active {
+      &:active, &.selected {
         background-color: $button-backgroundColor--subtle--active;
+        span {
+          &[class^="Spinner"] svg {
+            stroke: $base-white;
+          }
+        }
       }
-      &.danger {
+      &-link {
```

Comment:
when does this selector happen?

Reply:
this selector intended to handle the `appearance="subtle-link"` styling